packages/lib-user/src/components/UserHome/components/Dashboard/components/StatsTabs/StatsTabsContainer.js

```diff
@@ -2,49 +2,47 @@ import { shape, string } from 'prop-types'
 
 import { useStats } from '@hooks'
 import {
-  getDateInterval,
-  getDefaultDateRange,
-  getStatsDateString
+  getThisWeekDateRange
 } from '@utils'
 import StatsTabs from './StatsTabs'
 
-const formatStatsPreview = (thisWeekData, allTimeData) => {
+const formatStatsPreview = ({ allTimeData, thisWeekData }) => {
   return {
-    thisWeek: {
-      classifications: thisWeekData.total_count,
-      projects: thisWeekData.project_contributions?.length
-    },
     allTime: {
       classifications: allTimeData.total_count,
       projects: allTimeData.project_contributions?.length
+    },
+    thisWeek: {
+      classifications: thisWeekData.total_count,
+      projects: thisWeekData.project_contributions?.length
     }
   }
 }
 
-const DEFAULT_DATE_RANGE = getDefaultDateRange() // last 7 days
-
 export default function StatsTabsContainer({ user }) {
-  const thisWeekQuery = getDateInterval(DEFAULT_DATE_RANGE)
-  thisWeekQuery.project_contributions = true
-  const { data: thisWeekData } = useStats({
-    sourceId: user?.id,
-    query: thisWeekQuery
-  })
-
-  const allTimeQuery = getDateInterval({
-    endDate: getStatsDateString(new Date()), // Today's UTC date string like 2024-09-23.
-    startDate: getStatsDateString(user?.created_at) // Limit stats range to when the user acccount was created in panoptes.
-  })
-  allTimeQuery.project_contributions = true
+  const allTimeQuery = {
+    project_contributions: true
+  }
```

Comment:
Per https://github.com/zooniverse/eras/wiki/API-Callout-Examples#example-requests-2 ERAS defaults to all time so omitting the `start_date` and `end_date` I think still works as expected.

Reply:
That's good to know and cleans up this container quite a bit.